src/components/application_manager/include/application_manager/application.h

```diff
@@ -405,6 +405,20 @@ class Application : public virtual InitialApplicationData,
   virtual void UpdateHash() = 0;
 
   /**
+   * @brief Retrieves flag_sending_hash_change_after_awake_
+   * @return Returns TRUE if hashID was changed during suspended state
+   * otherwise returns FALSE.
+   */
+  virtual bool flag_sending_hash_change_after_awake() const = 0;
+
+  /**
+   * @brief Method is used when core receives OnAwakeSDL notification
+   * in order to change value of flag_sending_hash_change_after_awake_
+   * @param Contains FALSE
+   */
+  virtual void set_flag_sending_hash_change_after_awake(bool flag) = 0;
```

Comment:
@AKalinich-Luxoft 
Same note as above. Propose to change name to SetHashChangedAfterAwake()